app/services/add_ons/apply_taxes_service.rb

```diff
@@ -2,6 +2,8 @@
 
 module AddOns
   class ApplyTaxesService < BaseService
+    Result = BaseResult.define_with_attributes(:applied_taxes)
```

Comment:
nitpick: I'd rather have `Result = BaseResult[:applied_taxes]`   as syntax? (I think the `define_with_attributes is a bit much and shorter would be nicer)

Reply:
It's not nitpicking, it's important to find the best syntax 😆 That's why we love Ruby

Reply:
I agree, it's nicer, I will try it :+1: